code/fireball/fireballs.cpp

```diff
@@ -373,6 +373,90 @@ static void parse_fireball_tbl(const char *table_filename)
 				// rather than just having the default model that might or might not be used
 				fi->use_3d_warp = true;
 			}
+
+			if (optional_string("$Bobboau anim:")) {
```

Comment:
What do you think of renaming this and/or refactoring it slightly? I'd suggest "enhanced" as a name which offers a little more info over "bobboau" especially for people who have no idea who that is.

I also might recommend a slightly different parsing approach here and use `optional_string_one_of()` with "retail", "enhanced", and "cinematic" as options. Use that to set an `fi->style` to an enum class and then the `style` variable could be used everywhere else to gate behavior between the three styles. That also sets the code up nicely for future whoever to create a fourth style. Let me know if none of that makes any sense and I'll walk you through it in more detail.